CMakeLists.txt

```diff
@@ -54,6 +72,12 @@ link_directories(
   ${OFED_PREFIX}/lib
   ${LEVELDB_PREFIX}/lib
 )
+if(${FREEBSD})
```

Comment:
okay, if you insist. but better off understanding why before we blindly use `if(${FREEBSD})` all over the places.